server/server_media.go

```diff
@@ -65,6 +78,17 @@ func NewMediaServer(db *sql.DB, downloader *ipfs.Downloader, multiaccountsDB *mu
 	return s, nil
 }
 
+func (s *MediaServer) MakeBaseURL() *url.URL {
+	scheme := "https"
+	if UseHTTP {
+		scheme = "http"
+	}
+	return &url.URL{
+		Scheme: scheme,
+		Host:   s.mustGetHost(),
+	}
+}
+
```

Comment:
I don't know if I'm getting overly complex with this suggestion, but a `map[bool]string` could work here. 

```suggestion
func (s *MediaServer) MakeBaseURL() *url.URL {
	return &url.URL{
		Scheme: map[bool]string{true: "http", false: "https"}[UseHTTP],
		Host:   s.mustGetHost(),
	}
}
```
Or maybe?
```suggestion
func (s *MediaServer) MakeBaseURL() *url.URL {
	schemes := map[bool]string{true: "http", false: "https"}
	return &url.URL{
		Scheme: schemes[UseHTTP],
		Host:   s.mustGetHost(),
	}
}

```

To be honest feel free to completely ignore this suggestion. I think I'm just trying to make ternary expressions work in Go and Go doesn't do ternary expressions.

Reply:
I like the shorter version:
```golang
func (s *MediaServer) MakeBaseURL() *url.URL {
	return &url.URL{
		Scheme: map[bool]string{true: "http", false: "https"}[UseHTTP],
		Host:   s.mustGetHost(),
	}
}
```
😉